db/changes.go

```diff
@@ -495,6 +495,20 @@ func makeRevocationChangeEntry(logEntry *LogEntry, seqID SequenceID, channel cha
 	return entry
 }
 
+// GetDocVersionForAudit returns the document version for audit logging if we should log an audit event.
+func (ce *ChangeEntry) NeedsDocReadAudit() (string, bool) {
+	if ce.Err != nil {
+		return "", false
+	}
+	if ce.Deleted {
```

Comment:
I don't think we should have this deleted criteria here.  There are 1.x non-xattr cases that support document bodies on tombstones, and I also think that the `{"_deleted":true}` replacement body should be counted as a read.